google/cloud/aiplatform/training_jobs.py

```diff
@@ -294,10 +307,14 @@ def run(self) -> Optional[models.Model]:
     def _create_input_data_config(
         dataset: Optional[datasets._Dataset] = None,
         annotation_schema_uri: Optional[str] = None,
-        training_fraction_split: float = 0.8,
-        validation_fraction_split: float = 0.1,
-        test_fraction_split: float = 0.1,
+        training_fraction_split: Optional[float] = None,
```

Comment:
Moved default setting responsibility to server-side instead of in multiple places in the client.